### PR TITLE
Translations update from OSGeo Weblate

### PR DESCRIPTION
**Pull request by osgeoweblate**
# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [ ] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [ ] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [ ] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [ ] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [ ] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by jodygarnett** [comment]
I am not quite sure how to manage these...

**Comment by petersmythe** [comment]
My suggestion is that we gratiously accept these contributions via OSGeo Weblate, without verifying the actual translations and thus relying upon the Weblate service "blindly".

**Review by petersmythe — APPROVED** [review]

**Comment by aaime** [comment]
I'd tend to agree with @petersmythe , but is there anyone double checking that the translations are "sane" in weblate? 
At a minimum, I'd try to avoid jokes, swearwords and the like.

**Comment by petersmythe** [comment]
I believe that @AlexGacon is best positioned to advise us on what sort of quality checks https://weblate.osgeo.org/projects/geoserver has in place to prevent malicious translations.

**Comment by AlexGacon** [comment]
I can check it these evening. Normally, only registered users at osgeo can update the translations.

I didn't trigger this update.

**Comment by AlexGacon** [comment]
I checked : it seems to be an automatic trigger by weblate but I don't know why it pops up today.

**Comment by AlexGacon** [comment]
@petersmythe could you close this PR? I have reseted the weblate instance because of the changes it pushed to GH.

**Comment by AlexGacon** [comment]
PR updated with new French translation. Could someone accept it?

**Comment by jodygarnett** [comment]
I can, does this get backported also?

File: src/web/core/src/main/resources/GeoServerApplication_fr.properties [hunk 1]
```diff
@@ -1051,3 +1050,21 @@ PublishedConfigurationPage.cqlUsesInvalidAttribute=L'expression CQL source de l'
 PublishedConfigurationPage.attributeInvalidConversion=Un probl�me a �t� d�tect� dans l'attribut {0}, impossible de convertir depuis le type d'origine {1} vers le type cible {2}
 PublishedConfigurationPage.attributeInvalidCQL=CQL invalude pour la source {0}. {1}
 PublishedConfigurationPage.multiAttributeSameName=Plusieurs d�finitions ont �t� trouv�es pour l'attribut de sortie {0}
+AbstractWMSStorePage.headerName=Nom de l'ent\u00EAte HTTP
```
Comment (jodygarnett) [inline src/web/core/src/main/resources/GeoServerApplication_fr.properties:1053]: aside: These files can be setup to accept unicode if you change the file extension bit. Not sure if that works still with weblate?

**Pull request merged**